var/spack/repos/builtin/packages/py-saga-python/package.py

```diff
@@ -1,19 +0,0 @@
-# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
-# Spack Project Developers. See the top-level COPYRIGHT file for details.
-#
-# SPDX-License-Identifier: (Apache-2.0 OR MIT)
-
-from spack import *
-
-
-class PySagaPython(PythonPackage):
-    """A light-weight access layer for distributed computing infrastructure"""
-
-    homepage = "http://radical.rutgers.edu"
-    pypi = "saga-python/saga-python-0.41.3.tar.gz"
-
-    version('0.41.3', sha256='b30961e634f32f6008e292aa1fe40560f257d5294b0cda95baac1cf5391feb5d')
```

Comment:
@adamjstewart : Should this one be deprecated also instead of plain removing?  I though agree with @mtitov : the version is disfunct and should be removed ASAP.  But from you comment above it sounds like a deprecation period should be observed anyway?

Reply:
It's probably best to deprecate it. There may be someone who still has the old tarball in their downloads cache.

Reply:
ok, rolled it back with some changes (including deprecation status)